java/panama-bindings/generate-bindings.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
```

Comment:
```suggestion

set -e -u -o pipefail

```

Can you please make this stricter, so that we'll be notified via loud failures for things like undefined variables or commands that fail?

Reply:
This would also mean you can remove custom handling of return codes, like this stuff:

```shell
# Did Jextract complete normally? If not, stop and return
JEXTRACT_RETURN_VALUE=$?
if [ $JEXTRACT_RETURN_VALUE == 0 ]
then
  echo "Jextract SUCCESS"
else
  echo "Jextract encountered issues (returned value ${JEXTRACT_RETURN_VALUE})"
  exit $JEXTRACT_RETURN_VALUE
fi
```

Reply:
Hi @jameslamb, thank you for pointing that out. I completely agree with you. Just to clarify, I originally believed I could make the same change in my PR following this (so from my perspective, it wasn’t a matter of principle).